tests/atomistics/structure/test_sparse_list.py

```diff
@@ -44,6 +44,16 @@ def test__getitem__(self):
         self.assertEqual(self.cList[-1], 0)
         self.assertTrue(isinstance(self.cList[2:4], SparseList))
 
+        self.assertEqual(self.aList[[False, True, False, False, True, False, False]].list(),
+                         [True, True],
+                         "Indexing with boolean mask returns wrong values!")
+        cListModified = self.cList.__copy__()
+        cListModified[[True, True, False, False, False]] = 0.1
+        self.assertEqual(cListModified[0], 0.1,
+                         "Indexing with boolean mask sets wrong values!")
+        self.assertEqual(cListModified[1], 0.1,
+                         "Indexing with boolean mask sets wrong values!")
+
```

Comment:
Nitpick: For completeness why not check that any/all of the other indices aren't 0.1